distributions/otelcol-ebpf-profiler/manifest.yaml

```diff
@@ -0,0 +1,37 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/ebpf-profiler
+  name: otelcol-ebpf-profiler
+  description: OpenTelemetry Collector for eBPF Profiling
+  version: 0.123.1
+  output_path: ./_build
+
+exporters:
```

Comment:
The loadbalancing exporter speaks OTLP, should it be included?

Reply:
It does not seem to speak profiles